src/storage-blob-preview/azext_storage_blob_preview/_params.py

```diff
@@ -179,6 +182,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
 
     with self.argument_context('storage blob') as c:
         c.argument('blob_name', options_list=('--name', '-n'), arg_type=blob_name_type)
+        c.argument('destination_path', help='The destination path that will be appended to the blob name.')
+        c.argument('socket_timeout', deprecate_info=c.deprecate(),
+                   help='The socket timeout(secs), used by the service to regulate data flow.')
```

Comment:
If parameter `socket_timeout` is no longer needed and `deprecate_info` is added just to avoid breaking users, could we hide it?

Reply:
deprecate_info will not break customers from my understanding. I don't want to hide it because I want to let customer know it will be deprecated soon and remove the usage in scripts.

Reply:
I know there's no breaking change, I'm just not perfer to show a parameter which is deprecated for Track2. Because this field hide is more friendly to new users and reduces their learning costs.
However, displaying this field is better for users who write scripts to notice this change. But I personally think it's better to have the log output the deprecated information when executing the script, rather than let the users interactively execute `-h` command to find the deprecated information.

By the way, that's a small question and I can accept whether this parameter is displayed or not~

Reply:
Yes, I will hide it then.